doc/service_onboarding_guide.md

```diff
@@ -0,0 +1,24 @@
+Service Onboarding Guide
+======
+
+This article aims to provide a guide for service team to request a feature in Azure CLI.
+
+We encourage you to contact Azure CLI team as early as possible even if the service is not ready yet so that the work can be planned in advance. We officially support three kinds of communication channels, namely Github, email and Teams. The email address of Azure CLI is azpycli@microsoft.com
+
+The only mandatory action is opening an issue at [Service Team Support Request](https://github.com/Azure/azure-cli/issues/new?assignees=&labels=&template=Service_team_request.md&title=). Fill in the blanks of the issue template as detailed as possible to help Azure CLI people understand the feature request without difficulties and deliver the feature as expected.
```

Comment:
not only. CLI need Swagger/Service public.

Reply:
This sentence uses a rhetorical device of exaggeration. It emphasize that asking a feature request is easy. And I think they can open the issue in advance even swagger/service is not public to allow us to better arrange work. Of course, it is not that easy. I have listed 9 items of important information they'd better provide.

Reply:
I have updated it.